docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -214,7 +214,40 @@ For more information about devfile contents requirements, see the "What is outer
 +
 **Solution:** Select a predefined runtime type from the **Review and configure for deployment** view. Make sure you meet all of the {ProductName} <<repository requirements>> for the runtime you select, or configure your own <<custom build and deployment instructions>>.
 
-[role="_additional-resources"]
-.Additional resources
-For information about importing and configuring code to {ProductName} using the CLI, see link:https://redhat-appstudio.github.io/docs.stonesoup.io/Documentation/main/getting-started/getting_started_in_cli/#getting-started-in-the-cli[Getting started in the CLI
-].
\ No newline at end of file
+=== Creating a Red Hat Container Registry token
+
+To improve your application-building experience, configure a Red Hat Container Registry token.  
```

Comment:
Wonder if we wish to add a note on why a user might want to use the Red Hat Container Registry token ? 

Example, being able to use registry.redhat.io images as a base image in the Dockerfile - 

```
FROM registry.redhat.io/jboss-eap-7/eap74-openjdk11-openshift-rhel8
```

Reply:
totally agree, @sbose78